File: javascript/src/models/fifoEndpointConfigIn.ts
// this file is @generated

export interface FifoEndpointConfigIn {
  url: string;
  headers?: { [key: string]: string };
  key?: string | null;
}

export const FifoEndpointConfigInSerializer = {
  _fromJsonObject(object: any): FifoEndpointConfigIn {
    return {
      url: object["url"],
      headers: object["headers"],
      key: object["key"],
    };
  },

  _toJsonObject(self: FifoEndpointConfigIn): any {
    return {
      url: self.url,
      headers: self.headers,
      key: self.key,
    };
  },
};

File: kotlin/lib/src/main/kotlin/models/FifoEndpointConfigIn.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class FifoEndpointConfigIn(
    val url: String,
    val headers: Map<String, String>? = null,
    val key: String? = null,
)
